File: GEOEmergency_Final/DTO/EmergencyMessageDTO.cs
using System.ComponentModel.DataAnnotations;
using GEOEmergency.API.Models;

namespace GEOEmergency.DTOs
{
    public class EmergencyMessageDTO
    {
        [Required]
        public string Description { get; set; }
        
        [Required]
        public TargetDepartment TargetDepartment { get; set; }
        
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public string? Address { get; set; }
    }
}

File: emergency/src/main/java/com/emergency/dto/EmergencyMessageDTO.java
package com.emergency.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class EmergencyMessageDTO {
    private Long citizenId;
    @NotBlank
    private String description;
    private String targetDepartment = "BOTH";
    private Double latitude;
    private Double longitude;
    private String address;
}
